templates/meetup_user.html

```diff
@@ -0,0 +1,5 @@
+{% extends "social_user.html" %}
+
+{% block edit_profile %}
+<a href="https://meetup.com/profile/" target="_blank">Edit your Meetup.com profile</a>, enter it in the Bio field (without <code>www.</code>, then click here:
```

Comment:
minor nit: why without www? that depends on the user's web site, right?

Reply:
I'd had some issues with `webutil` parsing the incoming domain name as `jvt.me`, even though I'd set my URL in both Meetup and Bridgy as `jvt.me`.

That being said, it may have been that I didn't have both `www.jvt.me` and `jvt.me` set in my websites in Bridgy.. :thinking: As I guess it works as-is for Twitter? But Twitter renders it without `www.`, too :thinking:

Reply:
(Happy to remove it, or add a note elsewhere about it, just something interesting I found!)

Reply:
yeah bridgy etc deliberately strip leading `www.`, `m.`, etc. domain checking everywhere should still handle subdomains though, right?

Reply:
Good point - that makes me wonder if I got it to properly pull the URL from the `bio`, as I don't see any of my own code doing that :thinking: